src/github.com/matrix-org/dendron/dendron/main.go

```diff
@@ -125,7 +125,10 @@ func main() {
 		}
 	}
 
-	var synapseLog = log.WithField("synapse", synapseURL.String())
+	var synapseLog = log.WithFields(log.Fields{
+		"synapse": synapseURL.String(),
+		"app":     "synapse",
```

Comment:
I see what you did there :) but most of your log messages contextually include this already e.g:
```
Starting synapse
Error waiting for pusher
Failed to kill synchrotron
```

So it's already clear what is being operated on. It seems a bit pointless to include both forms, so if you're going down the `app` field route, you probably want to tidy up the log messages.